test/js/web/fetch/fetch-leak.test.js

```diff
@@ -1,98 +0,0 @@
-import { test, expect, describe } from "bun:test";
```

Comment:
@cirospaciari i deleted this file out of caution for having the same filename with different extensions